.travis.yml

```diff
@@ -1,2 +1,5 @@
 language: ruby
 jdk: oraclejdk9
+script:
+  - rake test
```

Comment:
Ok, with #10 merged, we use now bundle.... in order to use the dependencies for rake, we need to call it that way:

```
script:
  - bundle exec rake test
  - bundle exec rake check_style
```

It seems, that travis automatically uses bundle exec by default, if no custom script is configured, but now we have to do it manually.
This *should* fix the build....